### DISCUSSION TITLE
Can version 2.0 continue to use full memory mode?

### DISCUSSION THREAD

**Discussion by NTOHPN** [Chia-Network/bladebit#232] [Q&A] [answered]
In the help I don't see how to set 2.0 to full memory mode?

**Comment by NTOHPN** [comment]
Does anyone care?

**Comment by harold-b** [comment, marked as answer]
It was not available at the time of your question, but it is now in the [rc1 release](https://github.com/Chia-Network/bladebit/releases/tag/v2.0.0-rc1). You can use the `ramplot` command to plot in-memory in BB 2.
